sdl_android/src/main/java/com/smartdevicelink/api/BaseSubManager.java

```diff
@@ -0,0 +1,42 @@
+package com.smartdevicelink.api;
+
+import android.support.annotation.NonNull;
+
+import com.smartdevicelink.proxy.interfaces.ISdl;
+
+/**
+ * <strong>BaseSubManager</strong> <br>
+ *
+ * Note: This class is extended by SubManagers <br>
+ *
+ * It is broken down to these areas: <br>
+ *
+ * 1. <br>
+ */
+public abstract class BaseSubManager {
+
+	// states - if this gets more complicated we can move elsewhere
+	public int state, SETTING_UP = 0x00, READY = 0x30, SHUTDOWN = 0x60;
```

Comment:
The actual state constants need to be `public static final` and the state should be private.